### PR TITLE
Improved grid

### LINKED ISSUE #5: Mobile responsiveness
On mobile devices the home page doesn't fit well

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 22 - 12 - 23 

## Changes

- Add `class: "grid__text"` to modify on each grid box to modify the text more accurately
- Improve grid style with box-size and add hover effect with shadows

Screenshot 2023-12-22 110606
dows

### CODE REVIEW

File: CSS/home style/home_grid.css [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
 .grid__boxTitle {
-    font-size: 2.2rem;
-    padding-top: 1.5rem;
-    padding-bottom: 2rem;
+    font-size: 2.5rem;
+    margin: 1rem 0;
+}
+
+.grid__text {
+    font-family: 'Poppins', 'Arial';
```
Comment (vladiiancu) [inline CSS/home style/home_grid.css]: Where is this font defined?

**Review by vladiiancu — CHANGES_REQUESTED** [review]

File: CSS/home style/home_grid.css (hunk 1 above)
Comment (Alexandru-Bulai) [reply CSS/home style/home_grid.css]: True, that is not changing anything

Comment (vladiiancu) [reply CSS/home style/home_grid.css]: Well then... you either use the poppies font by inserting it on your website or you delete this entry

**Review by vladiiancu — CHANGES_REQUESTED** [review]
Resolve conflicts as well

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**